astronomer/providers/http/hooks/http.py

```diff
@@ -119,6 +119,7 @@ async def run(
                 )
                 try:
                     response.raise_for_status()
+                    await session.close()
```

Comment:
ideally yes even I tried to reproduce the same error but couldn't.
So added explicitly the close

Reply:
why would we add code for something which we couldnt reproduce?

Reply:
https://github.com/aio-libs/aiohttp/issues/5277#issuecomment-732243778 So the fix for this will be released in aiohttp 4.0 so the latest version is 3.8.4